config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml

```diff
@@ -31,7 +31,7 @@ presubmits:
     decorate: true
     branches:
     - main
-    - release/1.6
+    - release/1.7
```

Comment:
maybe just add 1.7 here and leave 1.6 as well..

Reply:
yes, we should leave 1.6 here